cypress/e2e/requests.cy.js

```diff
@@ -0,0 +1,22 @@
+describe("Requests Page", () => {
+  it("A logged out user should not able to see the requests list.", () => {
```

Comment:
can the `it` blocks be updated to flow like a sentence?

also, can you update to single quotes and no semi-colons to match the formatting of the rest of the app?